src/custom/utils/analytics/walletEvents.ts

```diff
@@ -7,3 +7,12 @@ export function changeWalletAnalytics(walletName: string) {
     label: walletName,
   })
 }
+
+type AddTokenActions = 'Success' | 'Fail'
```

Comment:
Nitpick: it's an action, so I'd prefer the past term rather than the substantive

```suggestion
type AddTokenActions = 'Succeeded' | 'Failed'
```